src/utils/sortedBy.js

```diff
@@ -0,0 +1,17 @@
+const sortedBy = (list, typeSorted) => {
+    let sortedList = list;
```

Comment:
you will mutate original list - you need to avoid this:)

Reply:
Sorry, I don't see it. It's a really fatal error. I changed this line for let sortedList = [...list]